modules/local/mhcnuggets_peptidesclass2pre.nf

```diff
@@ -23,13 +21,17 @@ process PREPROCESS_PEPTIDES_MHCNUGGETS_CLASS_2 {
     output:
         tuple val(meta), path("*_preprocessed_mhcnuggets_peptides"), emit: preprocessed
```

Comment:
Same as above, that doesnt fit to 
`def prefix = options.suffix ? "${meta.sample}_${options.suffix}" : "${meta.sample}_preprocessed_mhcnuggets_peptides"`
So i suggest
```suggestion
        tuple val(meta), path("*${prefix}"), emit: preprocessed
```
And that files has no other file ending? Seems strange. Isnt it maybe rather `*${prefix}*`?